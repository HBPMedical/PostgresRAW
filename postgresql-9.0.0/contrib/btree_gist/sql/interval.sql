-- interval check

CREATE TABLE intervaltmp (a interval);

\copy intervaltmp from 'data/interval.data'

SET enable_seqscan=on;

SELECT count(*) FROM intervaltmp WHERE a <  '199 days 21:21:23';

SELECT count(*) FROM intervaltmp WHERE a <= '199 days 21:21:23';

SELECT count(*) FROM intervaltmp WHERE a  = '199 days 21:21:23';

SELECT count(*) FROM intervaltmp WHERE a >= '199 days 21:21:23';

SELECT count(*) FROM intervaltmp WHERE a >  '199 days 21:21:23';

CREATE INDEX intervalidx ON intervaltmp USING gist ( a );

SET enable_seqscan=off;

SELECT count(*) FROM intervaltmp WHERE a <  '199 days 21:21:23'::interval;

SELECT count(*) FROM intervaltmp WHERE a <= '199 days 21:21:23'::interval;

SELECT count(*) FROM intervaltmp WHERE a  = '199 days 21:21:23'::interval;

SELECT count(*) FROM intervaltmp WHERE a >= '199 days 21:21:23'::interval;

SELECT count(*) FROM intervaltmp WHERE a >  '199 days 21:21:23'::interval;

