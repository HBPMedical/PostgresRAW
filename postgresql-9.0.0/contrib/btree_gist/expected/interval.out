-- interval check
CREATE TABLE intervaltmp (a interval);
\copy intervaltmp from 'data/interval.data'
SET enable_seqscan=on;
SELECT count(*) FROM intervaltmp WHERE a <  '199 days 21:21:23';
 count 
-------
   329
(1 row)

SELECT count(*) FROM intervaltmp WHERE a <= '199 days 21:21:23';
 count 
-------
   330
(1 row)

SELECT count(*) FROM intervaltmp WHERE a  = '199 days 21:21:23';
 count 
-------
     1
(1 row)

SELECT count(*) FROM intervaltmp WHERE a >= '199 days 21:21:23';
 count 
-------
   271
(1 row)

SELECT count(*) FROM intervaltmp WHERE a >  '199 days 21:21:23';
 count 
-------
   270
(1 row)

CREATE INDEX intervalidx ON intervaltmp USING gist ( a );
SET enable_seqscan=off;
SELECT count(*) FROM intervaltmp WHERE a <  '199 days 21:21:23'::interval;
 count 
-------
   329
(1 row)

SELECT count(*) FROM intervaltmp WHERE a <= '199 days 21:21:23'::interval;
 count 
-------
   330
(1 row)

SELECT count(*) FROM intervaltmp WHERE a  = '199 days 21:21:23'::interval;
 count 
-------
     1
(1 row)

SELECT count(*) FROM intervaltmp WHERE a >= '199 days 21:21:23'::interval;
 count 
-------
   271
(1 row)

SELECT count(*) FROM intervaltmp WHERE a >  '199 days 21:21:23'::interval;
 count 
-------
   270
(1 row)

